test/_payments-test.js

```diff
@@ -411,10 +411,8 @@ suite('payments', function() {
       .end(done);
```

Comment:
This might not be the best place to discuss it, but what is covered in _payments-test that isn't covered in payments-test? I recall that @geertweening said there's something in this test to keep.